pkg/conversion/converter.go

```diff
@@ -258,6 +263,22 @@ func (c *Converter) RegisterConversionFunc(conversionFunc interface{}) error {
 	return nil
 }
 
+func (c *Converter) RegisterFieldLabelConversionFunc(version string, conversionFunc interface{}) error {
```

Comment:
I think some cargo cult coding happened here-- there's no need to take an interface{} or use a reflect.Value here, since labels are a type known at compile time (map[string]string). :)